### ISSUE TITLE
Foundation uses CSS for ellipsis

### ISSUE THREAD

**Issue by TakteS** [mgwidmann/scrivener_html#43]
Hello!

When I have a lot of pages (1k for example), pagination looks like this:
![2016-12-28 17-45-38](https://cloud.githubusercontent.com/assets/7054944/21517847/117846a6-cd2e-11e6-99aa-af57e23ab314.png)

Passes (i.e. `...`) displays 2 times. Are it bug or feature?

If it can help, I do:

```
= pagination_links @subscribers, view_style: :foundation, query: @query
```

**Comment by mgwidmann** [comment]
Hmm, yeah it shouldn't appear twice... Can you write a failing test that would produce this issue? Is it only when there 1000+ pages?

**Comment by TakteS** [comment]
I checked just now and I have twice even if I have only 20 pages:
![2016-12-28 23-46-53](https://cloud.githubusercontent.com/assets/7054944/21524235/5278103c-cd60-11e6-9c4a-0f143d7ec899.png)

Here's versions of libraries I use:
```
"scrivener":  "2.1.1",
"scrivener_ecto": "1.0.3"
"scrivener_html": "1.4.0"
```

**Comment by TakteS** [comment]
Also I found that even if I do:

```
<%= Scrivener.HTML.pagination_links(%Scrivener.Page{total_pages: 100, page_number: 1}, view_style: :foundation) %>
```

then trouble keeps, but without `:view_style` option I have no twice:

```
<%= Scrivener.HTML.pagination_links(%Scrivener.Page{total_pages: 100, page_number: 1}) %>
```

Btw, I work with foundation 6, are you support this?

**Comment by mgwidmann** [comment]
Looks like it is a foundation problem. That makes this easier to find, thanks! I'll look into it in a little bit.

**Comment by TakteS** [comment]
@mgwidmann AFAIK it is because `ellipsis` class have `::after`option which automatically display `...` on page. So I just added option `ellipsis: ""` and it works fine

**Comment by TakteS** [comment]
Yes, I think so

**Comment by mgwidmann** [comment]
Need to fix this before the next release goes out. I'm going to try to get a PR opened tonight.

**Comment by mgwidmann** [comment]
Fixed in `v1.7.0`, please let me know if theres any issues. Thanks!
